package.json

```diff
@@ -79,6 +82,6 @@
     "eslint-plugin-react": "^7.19.0",
     "http-proxy-middleware": "^1.0.5",
     "react-axe": "^3.4.1",
-    "react-hook-form-devtools": "^1.2.0"
```

Comment:
This isn't in the existing package-lock.json
The new version listed doesn't exist in npm
@PanopticaRising please clarify what is correct